### ISSUE TITLE
iOS vulnerable to biometric bypass via "objection"

### ISSUE THREAD

**Issue by ssuppan** [smstuebe/xamarin-fingerprint#236]
My company did some pen testing on our Xamarin native app which is using plugin.fingerprint.
They were able to bypass biometric authentication via "objection v1.11.0".
This script/program allows a local user to hook into EvaluatePolicy() and EvaluateAccessControl().
When a bad fingerprint is scanned, you can return "true" instead of "false" and gain access to the app.

Supporting documentation can be found [here](https://github.com/sensepost/objection/wiki/Understanding-the-iOS-Biometrics-Bypass).

## Steps to reproduce

1. objection -g <app pkg name> explore

## Expected behavior
The objection script/program should not be able to bypass the bad fingerprint read

## Actual behavior
The objection script/program permits the pen tester to bypass fingerprint authentication

### Crashlog

If something causes an exception paste full stack trace + Exception here

## Configuration

**Version of the Plugin ** 2.1.5

**Platform:** iOS 12.X and greater

**Device:** iPhone 12

**Comment by smsissuechecker** [comment]
Hi @ssuppan,

I'm the friendly issue checker.
Thanks for using the issue template :star2:
I appreciate it very much. I'm sure, the maintainers of this repository will answer, soon.

**Comment by ssuppan** [comment]
This is not only an issue for fingerprint read on iOS.  I believe that Face ID is also vulnerable.

**Comment by jvillaro** [comment]
Yes this is an issue that we come to via the ethical hacking of our apps. I hope someone can help out

**Comment by MichaelLHerman** [comment]
While this is interesting, I don't think this is a security vulnerability. `objection` lets you (a security tester) bypass any biometric-authentication system dialog regardless of wether its accessed through this library, but the workflow for using `objection` relies on injecting code into the application package, resigning the IPA, and side-loading the application.  If a hacker had access to the device to do that, they might as well deploy a version of your application without a login screen.
